src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/components/content/main.js

```diff
@@ -34,13 +34,15 @@ define(function() {
         /** returns normalized event names */
         createEventName = function(postFix) {
             return namespace + postFix;
-        };
+        },
+
+        MEDIA_LANGUAGE = 'mediaLanguage';
```

Comment:
Didn't we usually use a `constants`-variable for this kind of stuff? This way it could be easily confused with events.